ElevatorSim/src/elevatorsim/scheduler/Scheduler.java

```diff
@@ -129,4 +156,12 @@ public void stopRunning() {
 	public SchedulerState getSchedulerState() { 
 		return state;
 	}
-}
+	
+	public ArrayList<ElevatorRequest> getStoredRequests(){
+		return this.storedRequests;
+	}
+	
+	public void setStoredRequests(ArrayList<ElevatorRequest> storedRequests) {
```

Comment:
I'm leery about having the ability to strip out and replace the entire arraylist. Why can't we have a addRequestToStorage or something like that instead?

Reply:
Added remove and add functions and removed this function